packages/components/avatar/src/avatar.ts

```diff
@@ -140,6 +147,13 @@ export class Avatar extends LitElement {
     this.borderWidth = this.#config?.badgeGapWidth * 2;
     this.setAttribute('shape', this.#config.shape);
 
+    this.#resizeObserver = new ResizeObserver(() => {
+      this.#checkOverflow();
+      return;
+    });
+
+    this.#resizeObserver.observe(this);
```

Comment:
I don't see a `.disconnect()` anywhere? Shouldn't that be done inside the `disconnectedCallback()`?

Reply:
Excellent point :)